nncf/parameters.py

```diff
@@ -83,3 +94,25 @@ def __init__(self,
         self.names = names
         self.patterns = patterns
         self.types = types
+
+
+def convert_ignored_scope_to_list(ignored_scope: Optional[IgnoredScope]) -> List[str]:
```

Comment:
```suggestion
def convert_ignored_scope_to_list(ignored_scope: Optional[IgnoredScope] = None) -> List[str]:
```

Reply:
ignored_scope does not have default value

Reply:
Ok, I got it.